Makefile

```diff
@@ -33,6 +33,10 @@ test_fmt:
 	section "Cargo fmt" ;\
 	cargo fmt -- --check
 
+test_commands:
+	cargo run --bin html5-parser-test >/dev/null
+	cargo run --bin parser_test >/dev/null
+
```

Comment:
While we're here, let's run these commands during the test run so that any errors that are introduced fail the CI run.  I was going to continue the pattern of the others, of including a `section "Commands"`, but for some reason it was swallowing nonzero exit codes for these commands.